Mention removing Spring Boot plugin (#488)
spring-cloud/spring-cloud-function: docs/src/main/asciidoc/adapters/gcp-intro.adoc

As of March 2020, Google Cloud Functions for Java is in Alpha.
You can get on the https://docs.google.com/forms/d/e/1FAIpQLScC98jGi7CfG0n3UYlj7Xad8XScvZC8-BBOg7Pk3uSZx_2cdQ/viewform[whitelist] to try it out.

> To deploy to Google Cloud Function, you need to produce a fat jar using the Shade plugin, rather than Spring Boot plugin.

Reviewer: ```suggestion
To deploy to Google Cloud Function, you need to produce a fat jar using the Shade plugin, rather than the Spring Boot plugin.
```

> First, *remove* Spring Boot plugin, e.g. remove this block:

Reviewer: ```suggestion
First, if you already have the Spring Boot plugin in your `pom.xml`, *remove* it:
```

[source, xml]
----

> <plugin>

Reviewer: Maybe show it as a commented out block?

Author: done!

    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-maven-plugin</artifactId>
</plugin>
----

> Then, *add( the Shade Plugin configuration to generate a fat jar when you run the `mvn package` command.

Reviewer: ```suggestion
Then, *add* the Shade Plugin configuration to generate a fat jar when you run the `mvn package` command.
```

</plugin>
----

> IMPORTANT: If both Spring Boot plugin and Shade plugin are present, Shade plugin may be shading a Spring Boot produced JAR, resulting a Fat JAR that's unusable in Google Cloud Function. Don't forget to remove the Spring Boot plugin!

Reviewer: Honestly, I haven't seen this. Is it also the case for AWS, which also uses Shade?

```suggestion
IMPORTANT: If both Spring Boot plugin and Shade plugin are present, Shade plugin may be shading a Spring Boot produced JAR, resulting in a Fat JAR that's unusable in Google Cloud Function. Don't forget to remove the Spring Boot plugin!
```
